### PR TITLE
feat: add avalanchego_http_allowed_hosts to reference

### LINKED ISSUE #55: Document avalanchego_http_allowed_hosts

### PR DESCRIPTION
**Pull request by leopaul36**
### Linked issues

- Fix #55 

### Dependencies

- https://github.com/AshAvalanche/ansible-avalanche-collection/pull/74

### Changes

- Add `avalanchego_http_allowed_hosts` to reference

### CODE REVIEW

File: docs/toolkit/ansible-avalanche-collection/reference/roles/avalanche-node.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ This Ansible role allows to bootstrap Avalanche nodes:
...
 | `avalanchego_http_port`               | [--http-port](https://docs.avax.network/build/references/avalanchego-config-flags#--http-port-int) argument                                                                                                                                                           | `9650`                                                                                                 |
+| `avalanchego_http_allowed_hosts`      | [--http-allowed-hosts](https://docs.avax.network/build/references/avalanchego-config-flags#--http-allowed-hosts-string) argument                                                                                                                                      | `localhost`                                                                                            |
```
Comment (Nuttymoon) [inline docs/toolkit/ansible-avalanche-collection/reference/roles/avalanche-node.md:30]: The field does not exist in the Avalanche docs. We should open a PR there.

Comment (leopaul36) [reply docs/toolkit/ansible-avalanche-collection/reference/roles/avalanche-node.md:30]: Yes I figured they will implement it afterwards but it looks like they still have not done it

Comment (leopaul36) [reply docs/toolkit/ansible-avalanche-collection/reference/roles/avalanche-node.md:30]: https://github.com/ava-labs/avalanche-docs/issues/1434

**Review by Nuttymoon — APPROVED** [review]

**Comment by leopaul36** [comment]
Rebased

**Pull request merged**